core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -559,27 +559,21 @@ void core_mmu_set_info_table(struct core_mmu_table_info *tbl_info,
 		tbl_info->num_entries = XLAT_TABLE_ENTRIES;
 }
 
-void core_mmu_create_user_map(struct tee_mmu_info *mmu, uint32_t asid,
-		struct core_mmu_user_map *map)
+void core_mmu_create_user_map(struct user_ta_ctx *utc,
+			      struct core_mmu_user_map *map)
 {
+	struct core_mmu_table_info dir_info;
+	vaddr_t va_range_base;
+	void *tbl = xlat_tables_ul1[thread_get_id()];
 
 	COMPILE_TIME_ASSERT(sizeof(uint64_t) * XLAT_TABLE_ENTRIES == PGT_SIZE);
 
-	if (mmu) {
```

Comment:
i guess this is called only for user TA, hence with a mmu.  should assert here that 'utc' is valid ?